Sources/Sentry/SentryHub.m

```diff
@@ -401,9 +401,6 @@ - (id _Nullable)getIntegration:(NSString *)integrationName
     return [integrations objectAtIndex:[integrations indexOfObject:integrationName]];
 }
 
-/**
- * Set global user -> thus will be sent with every event
- */
```

Comment:
just to confirm, we are removing from here because the public ones are under Sentry/Public right?

Reply:
Yes, no need to keep one in the implementation file.